src/renderer/components/layout/setting-layout.scss

```diff
@@ -10,7 +10,7 @@
   display: grid;
   color: var(--settingsColor);
   position: fixed;
-  z-index: 13!important;
+  z-index: var(--z-index-middle);
```

Comment:
Where is correlation between some `Layout` and visibility in the middle? Why the magic number was 13?

Reply:
Settings Layout being placed **above** main layout so it should have some bigger `z-index` number.

Number `13` previously used to overwrite specified z-indexes for hotbar items. Hotbar uses max value `12` (This is used to drag-n-drop purposes).

Adding some very low `z-index` + `isolation: isolate` over hotbar solves the issue.